lib/functions/lintly.sh

```diff
@@ -36,6 +36,11 @@ function AddLinterOptsForLintly() {
   [[ ! "${LINTER_OPTS[PYTHON_PYLINT]}" =~ "json" ]] && LINTER_OPTS[PYTHON_PYLINT]+=" --output-format=json"
   [[ ! "${LINTER_OPTS[DOCKERFILE_HADOLINT]}" =~ "json" ]] && LINTER_OPTS[DOCKERFILE_HADOLINT]+=" --format json"
   [[ ! "${LINTER_OPTS[TERRAFORM_TERRASCAN]}" =~ "json" ]] && LINTER_OPTS[TERRAFORM_TERRASCAN]+=" -o json"
+  # gitleaks --verbose conflicts with --quiet; remove these first
+  [[ ! "${LINTER_OPTS[GITLEAKS]}" =~ "-q" ]] && \
+      LINTER_OPTS[GITLEAKS]=${LINTER_OPTS[GITLEAKS]// --verbose/} && \
+      LINTER_OPTS[GITLEAKS]=${LINTER_OPTS[GITLEAKS]// -v/} && \
+      LINTER_OPTS[GITLEAKS]+=" --format=json --quiet"
```

Comment:
actually im wrong!

We do want `. --no-git` and. `--path=` but not just for lintly

Reply:
Also here we don't need --format=json , Just use this 
`gitleaks --path=. -q | lintly --format=gitleaks`

Reply:
According to its docs, it uses `--format=json` by default if omitted (I originally included it here to be explicit/in case they change the default in the future).

I've tried removing `--format=json`, and same json parsing issues (see my comments below):

[linter command](https://github.com/23andme-private/sarahc/runs/2437013866?check_suite_focus=true#step:4:1620)
```
2021-04-26 10:06:59 [DEBUG]   Running LintCodebase. FILE_TYPE: GITLEAKS. Linter name: gitleaks, linter command: gitleaks --config-path= --redact --no-git  --quiet, TEST_CASE_RUN: false, FILTER_REGEX_INCLUDE: , FILTER_REGEX_EXCLUDE:  files to lint: lint_test/test_gitleaks.py
```

[linter output](https://github.com/23andme-private/sarahc/runs/2437013866?check_suite_focus=true#step:4:1633) (note _not_ a json array, but just two objects concatenated together)
```
{"line":"    aws_access_key_id='REDACTED',","lineNumber":8,"offender":"REDACTED","commit":"","repo":"","repoURL":"","leakURL":"","rule":"AWS Access Key","commitMessage":"","author":"","email":"","file":".","date":"0001-01-01T00:00:00Z","tags":"key, AWS"}
{"line":"gh_pat = 'REDACTED'","lineNumber":13,"offender":"REDACTED","commit":"","repo":"","repoURL":"","leakURL":"","rule":"Github Personal Access Token","commitMessage":"","author":"","email":"","file":".","date":"0001-01-01T00:00:00Z","tags":"key, Github"}
```

[stack trace](https://github.com/23andme-private/sarahc/runs/2437013866?check_suite_focus=true#step:4:1663)
```
...
  File "/usr/local/lib/python3.7/site-packages/lintly/parsers.py", line 369, in parse_violations
    json_data = json.loads(output)
...
  File "/usr/local/lib/python3.7/json/decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 2 column 1 (char 255)
```